modules/nw-sriov-network-operator.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: REFERENCE
+[id="nw-sriov-network-operator_{context}"]
+= Example SR-IOV Network Operator for virtual functions
+
+You can use the Single Root I/O Virtualization (SR-IOV) Network Operator to allocate and configure Virtual Functions (VFs) from SR-IOV-supporting Physical Function NICs on the nodes.
+
+For more information on deploying the Operator, see _Installing the SR-IOV Network Operator_.
+For more information on configuring an SR-IOV network device, see _Configuring an SR-IOV network device_.
+
+There are some differences between running Data Plane Development Kit (DPDK) workloads on Intel VFs and Mellanox VFs. This section provides object configuration examples for both VF types.
+The following is an example of an `sriovNetworkNodePolicy` object used to run DPDK applications on Intel NICs:
+[source,yaml]
+----
+    apiVersion: sriovnetwork.openshift.io/v1
```

Comment:
I just noticed that this whole example is indented. Should the top level start with no indentation and everything else align from that? Please check with an SME or QE reviewer.

Reply:
None of the other code blocks have that extra tab, but checked with SME anyway. Removed.